src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -65,7 +74,40 @@ export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
         name={conversation.to.name}
         initials={conversation.to.initials}
       />
-      <Separator />
+      {item && (
+        <>
+          <Separator />
+          <Flex flexDirection="column" p={2}>
+            <Sans mb={2} size="3" weight="medium">
+              {item.__typename}
+            </Sans>
+            <Flex>
+              <Box height="60px" width="60px">
+                <ResponsiveImage src={item.image.thumbnailUrl} />
+              </Box>
+              <Flex flexDirection="column" ml={1}>
+                {item.__typename === "Artwork" && (
+                  <ArtworkDetails mt="-4px" artwork={item} />
```

Comment:
The margin here is weird, but the lineheight on the text in the artwork metadata pushes the body out of alignment with the image.